tests/core/test_utils.py

```diff
@@ -98,6 +98,7 @@ def test_float_conversion_to_decimal():
     d = {
         "int": -1,
         "float": 2.1,
+        "float_round": 0.918040672051180450807805755175650119781494140625,
```

Comment:
could you please to the test with an output of `time.time()`. I think that's the more realistic setting for the tracker store.